src/rdkafka_admin.c

```diff
@@ -838,6 +841,29 @@ rd_kafka_admin_worker (rd_kafka_t *rk, rd_kafka_q_t *rkq, rd_kafka_op_t *rko) {
 }
 
 
+typedef struct rd_kafka_AdminFanoutState_s {
+        rd_list_t requests;
```

Comment:
The `requests` list is not required by DeleteRecords, but we are using it in our implementation of DeleteGroups (which I'll be sending as soon as the `txns` branch lands).